### PR TITLE
streamline first-time login for admins 

### LINKED ISSUE #1386: post install setup needs to be more seamless
By design, new users who "just show up" after having authenticated with a third-party identity provider have no permissions. This is because not everyone known to whatever identity provider is in play necessary is authorized to do _anything_ in Brigade. Imagine a Brigade instance that uses GitHub for authentication. Anyone with a GitHub account can show up to the party, but naturally, we don't want randos showing up and automatically being granted permissions-- even read-only permissions.

This is all fine, except it creates a challenge for the very first user (presumably the person who installed Brigade) who must, currently, authenticate using the third-party identity provider, then re-authenticate as root, grant themselves admin privileges, then re-authenticate as themself.

This is further complicated by the fact that, imho, leaving root access enabled should be discouraged in a production cluster. This means the operator who handles the installation potentially has to install with root access enabled, then re-configure to disable root access after the first user has taken on the admin role.

We could make this much more seamless.

Many pieces of software I have worked with in the past automatically make the first user to register/login the admin. This would work, but I don't care for it because of the opportunity it presents for an unknown party to become the admin just by being first.

What I'd like to propose is that one or more admins can be specified by their ID (email address if using OpenID Connect or GitHub handle if using GitHub) at the time of installation (e.g. in `values.yaml`). When a new user authenticates, if they are matched to an ID in that list, they are automatically granted admin permissions.

### PR DESCRIPTION
**Pull request by krancour**
Fixes #1386

### CODE REVIEW

File: charts/brigade/values.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ apiserver:
   thirdPartyAuth:
     ## Valid values are "oidc" (for OpenID Connect), "github" (for OAuth2 with
     ## GitHub as the identity provider), or "disabled"
-    strategy: disabled
+    strategy: github
```
Comment (vdice) [inline charts/brigade/values.yaml:38]: I don't think we want these changes to go in, do we 😄 ?

I'd suggest leaving with `disabled` as the default still, but definitely keeping the new `admins` section/comments, perhaps with a sample commented-out GH username.

**Review by vdice — COMMENTED** [review]
I think this is a great way to address https://github.com/brigadecore/brigade/issues/1386.  Other than a comment around the chart values changes, looking good to me.

File: charts/brigade/values.yaml (hunk 1 above)
Comment (krancour) [reply charts/brigade/values.yaml:38]: Yep. That was an accident! Thanks for catching!

Comment (krancour) [reply charts/brigade/values.yaml:38]: Fixed after force push.

**Review by vdice — APPROVED** [review]

**Pull request merged**